ansible/roles/network/templates/network-manager-dnsmasq.conf

```diff
@@ -0,0 +1 @@
+address=/{{ liquid_domain }}/10.42.0.1
```

Comment:
Let's make a backlog issue for actually using the LAN ip config.

Reply:
Here you go. https://github.com/liquidinvestigations/setup/issues/75